homeassistant/components/teslemetry/strings.json

```diff
@@ -16,6 +16,110 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "backup_capable": {
+        "name": "Backup capable"
+      },
+      "charge_state_battery_heater_on": {
+        "name": "Battery heater"
+      },
+      "charge_state_charger_phases": {
+        "name": "Charger has multiple phases"
+      },
+      "charge_state_conn_charge_cable": {
+        "name": "Charge cable"
+      },
+      "charge_state_preconditioning_enabled": {
+        "name": "Preconditioning enabled"
+      },
+      "charge_state_scheduled_charging_pending": {
+        "name": "Scheduled charging pending"
+      },
+      "charge_state_trip_charging": {
+        "name": "Trip charging"
+      },
+      "climate_state_cabin_overheat_protection_actively_cooling": {
+        "name": "Cabin overheat protection actively cooling"
+      },
+      "climate_state_is_preconditioning": {
+        "name": "Preconditioning"
+      },
+      "components_grid_services_enabled": {
+        "name": "Grid services enabled"
+      },
+      "grid_services_active": {
+        "name": "Grid services active"
+      },
+      "state": {
+        "name": "Status"
+      },
+      "stream_automaticblindspotcamera": {
+        "name": "AutomaticBlindSpotCamera"
+      },
+      "stream_automaticemergencybrakingoff": {
+        "name": "AutomaticEmergencyBrakingOff"
+      },
+      "stream_blindspotcollisionwarningchime": {
+        "name": "BlindSpotCollisionWarningChime"
+      },
+      "stream_bmsfullchargecomplete": {
+        "name": "BmsFullchargecomplete"
+      },
+      "stream_brakepedal": {
+        "name": "BrakePedal"
```

Comment:
Some of these strings are like this with no space between the words and others have spaces. Consider making them all consistent.  And this one should be "Brake Pedal"

Reply:
They have all been removed, they related to a future feature and should not have been included.